tests/e2e/test_install.go

```diff
@@ -0,0 +1,78 @@
+// Copyright © 2023 Cisco Systems, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = When("Installing Koperator", Ordered, func() {
+	var kubeconfigPath string
+	var kubecontextName string
+
+	It("Acquiring K8s config and context", func() {
+		var err error
+		kubeconfigPath, kubecontextName, err = currentEnvK8sContext()
+
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	kubectlOptions := kubectlOptions(kubecontextName, kubeconfigPath, "")
+
+	When("Installing cert-manager", func() {
+		It("Installing cert-manager CRDs", func() {
+			certManagerCRDPath, err := certManagerHelmDescriptor.crdPath()
+			Expect(err).NotTo(HaveOccurred())
+
+			err = installCRDs(kubectlOptions, certManagerCRDPath)
+			Expect(err).NotTo(HaveOccurred())
+		})
```

Comment:
I think it is, I will try this.

Reply:
Fixed in 87b788f.
Left the now unused stuff in for now because I feel like we might need it soon and don't want to reimplement, but if we finish the next batch of test cases and don't need it then we can throw those out.